infra/serverless/providers.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region  = "eu-west-2"
```

Comment:
Is the region here for specific regional based pricing data or simply for access to the pricing API?

Reply:
This is the region to set up the infrastructure in I believe